feat(reporter): expose worker info on fixture teardown errors (#40444)
microsoft/playwright: docs/src/test-reporter-api/class-reporter.md

The error.

### param: Reporter.onError.workerInfo
* since: v1.60
- `workerInfo` ?<[WorkerInfo]>

> When the error originates from a worker fixture teardown, contains information about the worker that produced it. `undefined` for errors that are not associated with a specific worker.

Reviewer: ```suggestion
Contains information about the worker that produced this error. `undefined` for errors that are not associated with a specific worker.
```

Author: Applied the suggestion. Pushed in e79315b36.
